terraform/application/application.tf

```diff
@@ -56,6 +56,6 @@ module "worker_application" {
   cluster_configuration_map  = module.cluster_data.configuration_map
   kubernetes_config_map_name = module.application_configuration.kubernetes_config_map_name
   kubernetes_secret_name     = module.application_configuration.kubernetes_secret_name
-
+  replicas                   = var.sidekiq_replicas
```

Comment:
Sidekiq isn't used, this should be var.worker_replicas

Reply:
changed to worker_replicas